Buildings/Experimental/DHC/EnergyTransferStations/Cooling/DirectControlled.mo

```diff
@@ -0,0 +1,300 @@
+within Buildings.Experimental.DHC.EnergyTransferStations.Cooling;
+model DirectControlled "Direct cooling ETS model for district energy systems with in-building 
+  pumping and deltaT control"
+  extends
+    Buildings.Experimental.DHC.EnergyTransferStations.BaseClasses.PartialETS(
+    final typ=DHC.Types.DistrictSystemType.Cooling,
+    final have_weaBus=false,
+    final have_chiWat=true,
+    final have_heaWat=false,
+    final have_hotWat=false,
+    final have_eleHea=false,
+    final nFue=0,
+    final have_eleCoo=false,
+    final have_pum=false,
+    final have_fan=false,
+    nPorts_aChiWat=1,
+    nPorts_bChiWat=1);
+  // Mass flow rates
+  parameter Modelica.Units.SI.MassFlowRate mDis_flow_nominal(
+    final min=0,
+    final start=0.5)
+    "Nominal mass flow rate of district cooling side"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.MassFlowRate mBui_flow_nominal(
+    final min=0,
+    final start=0.5)
+    "Nominal mass flow rate of building cooling side"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.MassFlowRate mByp_flow_nominal(
+    final min=0,
+    final start=0.5)
+    "Nominal mass flow rate through the bypass segment"
+    annotation(Dialog(group="Nominal condition"));
+  // Pressure drops
+  parameter Modelica.Units.SI.PressureDifference dpConVal_nominal(
+    final min=0,
+    displayUnit="Pa")=6000
+    "Nominal pressure drop in the control valve"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.PressureDifference dpCheVal_nominal(
+    final min=0,
+    displayUnit="Pa")=6000
+    "Nominal pressure drop in the check valve"
+    annotation(Dialog(group="Nominal condition"));
+  // Controller parameters
+  parameter Real k(
+    final min=0,
+    final unit="1")=1
+    "Gain of controller"
+    annotation (Dialog(group="PID controller"));
+  parameter Modelica.Units.SI.Time Ti(
+    final min=Modelica.Constants.small)=120
+    "Time constant of integrator block"
+    annotation (Dialog(group="PID controller",enable=controllerType == Buildings.Controls.OBC.CDL.Types.SimpleController.PI or controllerType == Buildings.Controls.OBC.CDL.Types.SimpleController.PID));
+  parameter Modelica.Units.SI.Time Td(
+    final min=0)=0.1
+    "Time constant of derivative block"
+    annotation (Dialog(group="PID controller",enable=controllerType == Buildings.Controls.OBC.CDL.Types.SimpleController.PD or controllerType == Buildings.Controls.OBC.CDL.Types.SimpleController.PID));
+  parameter Real yMax(
+    final start=1)=1
+    "Upper limit of output"
+    annotation (Dialog(group="PID controller"));
+  parameter Real yMin=0.01
+    "Lower limit of output"
+    annotation (Dialog(group="PID controller"));
+  parameter Modelica.Blocks.Types.SimpleController controllerType=Modelica.Blocks.Types.SimpleController.PI
```

Comment:
move the the `controllerType` parameter before the parameter `k`, so that the user firstly choose the controller type, then set the `k`, (and `Ti` if it is PI controller), and (`Td` if it is PID or PD controller).

Reply:
Thanks for the suggestions! I have changed the order for this model and another model with the same issue.